### DISCUSSION TITLE
Will MPCTEMP be included in an update?

### DISCUSSION THREAD

**Discussion by salomonje** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2717] [Ideas]
Marlin 2.0.9.4 introduced the new Predictive Temp tuning model, which is still experimental.   
In Marlin mode of the TFT35 there are 2 options available , MPC Edit and MPC Auto Tune.
Is there any chance that it will be included in any version in the future?

**Comment by kisslorand** [comment]
>Is there any chance that it will be included in any version in the future?

![MPC_1](https://user-images.githubusercontent.com/50251547/219008173-fab0b413-ab1a-44c9-93f7-f5126c953258.jpg)
![MPC_2](https://user-images.githubusercontent.com/50251547/219008178-001bf1da-964b-4f00-aaca-5ffd73b767ab.jpg)
![MPC_3](https://user-images.githubusercontent.com/50251547/219008179-bc216dba-9a0b-4921-ad9b-e7e458403a35.jpg)

What is it exactly that should be included in addition to these?

**Reply by salomonje** [reply]
Nothing ;-) Problem:  my looks a bit different.  
![image](https://user-images.githubusercontent.com/100471790/219010710-89177faa-2cf1-43fa-927d-14f027971e1c.png)

**Reply by kisslorand** [reply]
In this case make sure you are on the latest build.
https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/tree/master/Copy%20to%20SD%20Card%20root%20directory%20to%20update

**Reply by salomonje** [reply]
Thanks,
installed the update and started MPC

**Comment by iduarte1966** [comment]
Se iniciar o MPC por M306 T funciona, se começar pelo screentouch fica sempre em "Busy processing"
